test/Microsoft.TestPlatform.CommunicationUtilities.PlatformTests/SocketServerTests.cs

```diff
@@ -35,9 +35,11 @@ public void Dispose()
 #if NETFRAMEWORK
             // tcpClient.Close() calls tcpClient.Dispose().
             this.tcpClient?.Close();
+            GC.SuppressFinalize(this);
```

Comment:
see the next comment.

Reply:
resolved added GC.SuppressFinalize(this); after the #endif